style.css

```
@@ -144,14 +144,29 @@ body {
 }
 
 .note {
+  animation: fadein 0.7s;
```

```suggestion
  animation: fadein 0.3s;
```
Decrease this to 0.3 seconds as this speed up the total time.

---

I've completed the task. I hope its all good now.